client/src/components/DeleteUserModal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Button, Modal } from 'semantic-ui-react';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+	onClose: () => void;
+	deleteUser: () => void;
+}
+
+const DeleteUserModal: React.FC<Props> = (props) => {
+	const { deleteUser, onClose } = props;
```

Comment:
you can destructure arguments straight in func declare
const DeleteUserModal = ({deleteUser, onClose}:props) => ...

Reply:
Roman asked destructure props in function body

Reply:
ok )